package.json

```diff
@@ -6,13 +6,14 @@
   "keywords": [],
   "scripts": {
     "test": "nyc mocha -t 30000 --require source-map-support/register build/test/*.js",
-    "test-one": "nyc mocha -t 30000 --require source-map-support/register $1",
+    "test-one": "nyc mocha -t 30000 --require source-map-support/register --grep 'packs a custom file' $1",
```

Comment:
I think the `--grep` should be removed so all tests are tested before this lands.

Reply:
great catch =)